src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_help.py

```diff
@@ -0,0 +1,72 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.help_files import helps
+
+helps['cognitiveservices list'] = """
+    type: command
+    short-summary: list all the existing cognitive services accounts under a resource group or current azure subscription
+    examples:
+        - name: list all the cognitive services accounts in a resource group 
+          text: az cognitiveservices list -g MyResourceGroup
+"""
+
+#helps['cognitiveservices checkskuavailability'] = """
+#    type: command
+#    short-summary: check the available skus
+#"""
```

Comment:
Do you mean the command is obsoleted or the help doc is obsoleted? Either way, this should be removed from the source code.